src/file/file.repository.ts

```diff
@@ -20,61 +22,141 @@ const sort = {
  */
 export default class FileRepository {
   /**
-   * Adds a given file to the DB.
+   * Add a given file to the DB.
    * @param file - is the file to be added to the DB
    */
-  static create(file: IFile): Promise<IFile> {
+  static create(file: any): Promise<IFile> {
     file.parent = file.parent ? new ObjectID(file.parent) : null;
     return fileModel.create(file);
   }
 
   /**
-   * Updates the file metatata by its id.
+  * Add a given shortcut file to the DB.
+  * @param file - is the shortcut file to be added to the DB
+  */
+  static async createShortcut(file: any): Promise<IFile> {
+    const populatedShortcut = await this.baseFileToIFile(await shortcutModel.create(file));
+    const shortcutAsFile: IFile = { ...(populatedShortcut.fileID as any), ...populatedShortcut };
```

Comment:
Why aren't you using your function `populatedShortcutToFile`?

Reply:
we agreed on doing this i remember.

Reply:
done, true.